TicTacToe/src/com/ticTacToe/GridService.java

```diff
@@ -0,0 +1,112 @@
+package com.ticTacToe;
+
+import com.ticTacToe.model.Player;
+import com.ticTacToe.model.TicTacToeGrid;
+
+import java.util.List;
+
+public class GridService {
+    private String[][] grid;
```

Comment:
Is this required? If we already keeping ticTacToeGrid, we can get the 2d array at runtime.